data-science-onramp/data-cleaning/clean.py

```diff
@@ -180,7 +178,7 @@ def compute_end_udf(duration, start, end):
         df = df.withColumn(name, obj["udf"](*obj["params"]))
 
     # Display sample of rows
-    df.sample(False, 0.001).show(n=100)
+    df.sample(False, 1.).show(n=100)
```

Comment:
Is it necessary for order of the data to be randomized? Also, perhaps it would be better if it's made explicit for the maintainers that this line is necessary for testing.

Reply:
Good point